linters/eslint/index.js

```diff
@@ -25,5 +25,26 @@ module.exports = {
                 };
             },
         },
+        'document-active-element': {
```

Comment:
Lint for `document.activeElement` to prevent bugs from slipping back in in the future.